openmetadata-core/src/main/resources/json/schema/entity/services/connections/metadata/openMetadataConnection.json

```diff
@@ -14,6 +14,11 @@
     }
   },
   "properties": {
+    "clusterName": {
```

Comment:
do we need this from connection standpoint?

Reply:
As discussed offline, we need to pass that parameter in the ingestion pipeline so the Airflow instance knows the `clusterName` and can build the right secret id. I prefer to handle adding the `identity` to the secret id in [this](https://github.com/open-metadata/OpenMetadata/issues/6615) task.